src/test/fuzz/transaction.cpp

```diff
@@ -102,7 +102,14 @@ FUZZ_TARGET_INIT(transaction, initialize_transaction)
     (void)AreInputsStandard(tx, coins_view_cache, true);
     (void)IsWitnessStandard(tx, coins_view_cache);
 
+    bool block_hash;
+    bool include_hex;
+    try {
+        ds >> block_hash;
+        ds >> include_hex;
+    } catch (const std::ios_base::failure&) {
+        return;
```

Comment:
If you initialize the bools to `false`, you can remove the early return (This is also what FuzzedDataProvider would do)

Reply:
That's a good idea. Ok, one sec